spec/controllers/custom_authorities_controller_spec.rb

```diff
@@ -195,21 +195,21 @@
     end
   end # query_users
 
-  describe '#lcsh_names', :vcr do
+  describe '#lcnaf_names', :vcr do
```

Comment:
Updating this spec for renamed method and route.

Reply:
Great! Thanks for doing this.